File: kotlin_android_dev/app/src/main/kotlin/com/kotlin_android_dev/named_utility/FactoryModelWrapperRepositoryUtility.kt
package com.kotlin_android_dev.named_utility

import com.kotlin_android_dev.model_wrapper_repository.example_wrapper_repository.ExampleWrapperRepository

class FactoryModelWrapperRepositoryUtility private constructor() {
    companion object {
        fun getExampleWrapperRepository(): ExampleWrapperRepository {
            return ExampleWrapperRepository()
        }
    }
}

File: dart_mobile_dev/lib/named_utility/factory_model_wrapper_repository_utility.dart
import 'package:meta/meta.dart';
import 'package:dart_mobile_dev/model_wrapper_repository/example_wrapper_repository/example_wrapper_repository.dart';

@immutable
final class FactoryModelWrapperRepositoryUtility {
  const FactoryModelWrapperRepositoryUtility._();

  static ExampleWrapperRepository get getExampleWrapperRepository {
    return const ExampleWrapperRepository();
  }
}